src/resources/js/tickets.js

```diff
@@ -637,16 +637,21 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 
 		$tribe_tickets.trigger( 'pre-save-ticket.tribe', e );
 
-		var $orders = $base_panel.find( '.tribe-ticket-field-order' );
+		var $ticketID = $edit_panel.find( '#ticket_id' ).val();
+		var $editParent = $base_panel.find( `[data-ticket-type-id="${$ticketID}"]` );
```

Comment:
We should be ready now to use `let` and/or `const`

Reply:
that makes sense but i figured it was better to keep the files themselves consistent. So use `let` and `const` in v2 files but existing v1 files, use whatever the file is currently using.

Let me know if you want me to update this anyways